package.json

```diff
@@ -146,10 +146,11 @@
     "csv-validator": "^0.0.3",
     "devtools-protocol": "0.0.939404",
     "es-main": "^1.0.2",
-    "eslint": "^7.23.0",
-    "eslint-config-google": "^0.9.1",
-    "eslint-plugin-import": "^2.24.2",
-    "eslint-plugin-local-rules": "0.1.0",
+    "eslint": "^8.4.1",
+    "eslint-config-google": "^0.14.0",
+    "eslint-formatter-codeframe": "^7.32.1",
```

Comment:
what does this do?

Reply:
![image](https://user-images.githubusercontent.com/39191/145650306-87784a4b-0ab2-44f5-a267-4c9314c3dd53.png)

we have it configured so the codeframe formatter is used in CI so if it fails there we see exactly what the prob is in the logs.

i also prefer using it locally. 

in latest eslint they extracted the formatter so its not shipped in the main pkg anymore.